src/engraving/libmscore/textbase.cpp

```diff
@@ -1709,7 +1709,7 @@ TextBase::~TextBase()
 
 void TextBase::drawSelection(mu::draw::Painter* p, const RectF& r) const
 {
-    mu::draw::Brush bg(QColor("steelblue"));
+    mu::draw::Brush bg(mu::draw::Color("steelblue"));
```

Comment:
I believe this color needs to be moved to the configuration
(and not use named color)